eel/eel-editable-label.c

```diff
@@ -1339,9 +1345,18 @@ get_layout_location (EelEditableLabel  *label,
 #endif
     GtkAllocation allocation;
 
+#if !GTK_CHECK_VERSION (3, 0, 0)
     misc = GTK_MISC (label);
+#endif
     widget = GTK_WIDGET (label);
+#if GTK_CHECK_VERSION (3, 0, 0)
```

Comment:
All ```margin_*``` variables are assigned here, but in the block a few lines below they're assigned once again, so the values assigned here aren't used at all...